news/2024/2024-06-19-osu-catch-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,110 @@
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2172850090&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+We started off this weekend with losses from Australia, Indonesia and Japan, knocking all three teams out of the tournament and all three of them having a tied placement of 9th–12th. This made the Russian Federation the last Asian hope to place themselves for the top 8 in this years' World Cup.
+
+Seed 8 and seed 10 of this CWC started the match on [MM1](https://osu.ppy.sh/beatmapsets/2195217#fruits/4644675) where Peru confidently got a first point on the board on the back of a huge score on the Hard Rock slot of [Lunati1kid](https://osu.ppy.sh/users/25723773). After two wins from the Russian Federation on [NM1](https://osu.ppy.sh/beatmapsets/2195010#fruits/4644233) and [NM2](https://osu.ppy.sh/beatmapsets/2195014#fruits/4644238) and two more wins from Peru on [MM2](https://osu.ppy.sh/beatmapsets/2195061#fruits/4644324) and [MM3](https://osu.ppy.sh/beatmapsets/2195058#fruits/4644312), Peru were able to get a breakpoint on [HR1](https://osu.ppy.sh/beatmapsets/2195113#fruits/4644437) with an impressive full combo from [Lunati1kid](https://osu.ppy.sh/users/25723773) ([00:37:26](https://www.twitch.tv/videos/2172850090?t=0h37m26s)).
+
+[Lunati1kid](https://osu.ppy.sh/users/25723773) has been one of the most promising players in recent years and he definitely lived up to the expectations as he secured another point for Peru with a full combo on [DT1](https://osu.ppy.sh/beatmapsets/2195045#fruits/4644298). Russian Federation were able to claw one point back on [NM4](https://osu.ppy.sh/beatmapsets/2195020#fruits/4644247) before Peru closed out the match 7 to 3 with wins on [DT3](https://osu.ppy.sh/beatmapsets/2195049#fruits/4644303) and [DT4](https://osu.ppy.sh/beatmapsets/2195051#fruits/4644305) ([01:09:43](https://www.twitch.tv/videos/2172850090?t=1h9m43s)).
+
+Peru advanced to the top 8, marking this the best year in their CWC history. However, in their second match of the weekend, they fell short to Canada locking them in the 7th–8th slot for this year. A beautiful performance from a team that wasn't able to win a match last year; there is a very bright future ahead of the Peruvian squad.
+
+The Russian Federation ends up finishing 9th–12th and this means there are no Asian teams making the top 8. This in its own is already a piece of history, as there has never been an edition where an Asian team has not finished on the podium of the osu!catch World Cup. And it gets even crazier, as the last time a non-Asian team won CWC was Chile in 2013 (!). Unless Chile pulls off an insane run in the loser bracket, chances of getting a new winner increase after every weekend. 2024 has already proven to be one of the most memorable editions in CWC history, and it doesn't stop here.
```

Comment:
maybe it'd be less scuffed to just bold this text instead of the exclamation mark
```suggestion
The Russian Federation ends up finishing 9th–12th and this means there are no Asian teams making the top 8. This in its own is already a piece of history, as there has never been an edition where an Asian team has not finished on the podium of the osu!catch World Cup. And it gets even crazier, as the last time a non-Asian team won CWC was Chile in **2013**. Unless Chile pulls off an insane run in the loser bracket, chances of getting a new winner increase after every weekend. 2024 has already proven to be one of the most memorable editions in CWC history, and it doesn't stop here.
```

Reply:
it's a stylistic thing, could go with either